clients/drcachesim/scheduler/scheduler_impl.h

```diff
@@ -206,14 +204,25 @@ template <typename RecordType, typename ReaderType> class scheduler_impl_tmpl_t
             queued_record_t(const RecordType &record, uint64_t next_trace_pc)
                 : record(record)
                 , next_trace_pc(next_trace_pc)
+                , next_trace_pc_valid(true)
             {
             }
 
             RecordType record;
+            // If this is zero, it means the next trace pc should be obtained
+            // from the input stream get_next_trace_pc() API instead.
```

Comment:
Maybe I caught this mid-update: I see a new valid field below, which I believe makes this comment stale?